### ISSUE TITLE
How Does Green Tea Help Weight Loss?

### ISSUE THREAD

**Issue by nadjibandnaaima** [nadjibandnaaima/diets-for-quick-weight-loss#566]
<div dir="ltr" style="text-align: left;">For many people, finding a weight loss supplement that&rsquo;s safe and effective would be nothing short of a miracle. We all know that there are no magic bullets when it comes to weight loss; we have to consume fewer calories in a day than our body uses in order to lose weight. However, finding ways to burn fat faster would speed up the process and make it easier.<br><br>
One product that seems to have much promise in helping people to lose weight is green tea. In some studies, green tea has been shown to help regulate insulin and lower cholesterol and triglycerides. In addition, it is believed that green tea speeds up the metabolism, so that we burn fat faster, and can even turn off the receptors that make us want to eat.<br><br>
In addition, green tea, unlike many other weight loss aids is very healthy overall. Green tea is a potent source of anti-oxidants, which help us stay healthy and disease free by fighting free radicals. Free radicals are oxygen containing molecules that are a by-product of our digestive processes. Left unchecked, they damage our cells and DNA, eventually leading to disease.<br><br>
However, green tea and other foods with anti-oxidants keep free radicals at bay, preventing aging and disease. For example, diets rich in green tea and other anti-oxidants have been linked to a reduction in the risk of heart disease and cancer. So, green tea is a weight loss supplement that you can feel good about, because it can improve your overall health while it helps you shed the pounds.<br><br>
Most of us are aware of the problems that many other weight loss supplements can cause. Ephedra, for example, has proven to be an effective way to speed up the metabolism, but can have very dangerous side effects. In fact, it has even been linked to heart attacks and death in some users.<br><br>
But, just how does green tea work?&nbsp; There are several components in green tea that seem to work together to provide the health and weight loss benefits that green tea&rsquo;s so famous for.&nbsp;&nbsp; Researchers have pinpointed three components in green tea that seem to have an effect on weight loss.<br><br>
Caffeine<br><br>
Green tea has lower caffeine content than coffee or black tea, but it does contain caffeine. Interestingly enough, however, green tea&rsquo;s caffeine does not seem to have the negative side effects found in other products containing caffeine. Green tea doesn&rsquo;t cause jitters or speed up the heart rate the way that other caffeinated beverages can. However, the caffeine in green tea does appear to speed up the metabolism.<br><br>
Many of us take our caffeine intake for granted, but caffeine has been shown to aid in weight loss. Many studies have shown that people on a low calorie diet who consume caffeine daily lose more weight overall than those who are on a low calorie diet alone.<br><br>
Catechins<br><br>
Catechins are the anti-oxidants in green tea. Catechins are polyphenolic antioxidant plant metabolites, specifically flavonoids called flavan-3-ols. They are found in many plants, including the cocoa plant and the coffee plant, but the best source of catechins is the Camellia sinensis (tea) plant.<br><br>
The reason that green tea is considered a better source of catechins than black tea is because of the difference in processing. Black tea is fermented, which changes the catechins. Green tea, however, is unfermented, leaving the catechins in their natural state.<br><br>
Because tea catechins are some of the most powerful anti-oxidants around, they are some of the most effective at preventing disease. This is why diets rich in tea are thought to prevent many diseases, including heart disease and cancer.<br><br>
In terms of weight loss, catechins are believed to suppress lipid (fat) absorption from the diet. This is important because it helps reduce cholesterol, but it is also thought to prevent the accumulation of body fat.<br><br>
Theanine<br><br>
Theanine is the main amino acid in green tea. Theanine may be important to weight loss because of its ability to induce relaxation and the release of dopamine. Dopamine is a chemical produced by the brain that helps in providing a feeling of well being in the body.<br><br>
Theanine also seems to minimize the effects of caffeine, which is why the caffeine in green tea doesn&rsquo;t seem to cause jitters and an increased heart rate the way that caffeine in other beverages can. So, while theanine may not actively cause weight loss, it can be effective in creating a feeling of calm in those who ingest it. This power may help those who overeat as a result of stress to curb this tendency.<br><br>
It is certain that more studies are needed on how green tea helps with weight loss. One interesting study on the subject, by the Shizuoka University in Japan, evaluated all three of these green tea components individually, as well as in combination, to try and determine just what components are most important.<br><br>
This study concluded that the most important component of green tea for weight loss is caffeine. Caffeine is responsible for speeding up the metabolism and creating thermogenesis, which burns fat. However, the study also showed that catechins do seem to have an anti-obesity effect on their own, helping the body to eliminate dietary fat rather than absorbing it.<br><br>
In short, green tea seems to be the near perfect combination for helping lose weight and improve health at the same time. Diet and exercise are still critical to taking off excess weight and keeping it off for good, but it does appear that green tea can be an effective tool in fighting the battle of the bulge.<br><br><br>
</div>
<br><br>
via diets for quick weight loss http://ift.tt/1RvztHh<br>
December 19, 2015 at 04:11PM
